src/commands/PingCommand.ts

```diff
@@ -16,11 +17,22 @@ export class PingCommand implements ICommand {
     const greetings = [
       'Aye', "What's up",
       'Hi', 'Hello',
-      'Hiya', 'Hola'
+      'Hiya', 'Hola',
+      'Hi there', 'Fakka',
+      'Здравствуйте', '你好'
     ];
 
-    const { displayName } = message.guild.member(message.author);
-    const index = Math.floor(Math.random() * greetings.length);
-    await message.channel.send(`${greetings[index]} ${displayName}!`);
+    const i = Math.floor(Math.random() * greetings.length);
+
+    const pingEmbed: MessageEmbedOptions = {
+      title: `${greetings[i]} ${message.author.username}!`,
```

Comment:
```suggestion
      title: `${greetings[i]} ${message.member.displayName}`,
```

Reply:
@stingalleman You should see @stijnvdkolk's revision instead. This is way simpler than my change proposal. ;)